datafusion/functions-aggregate/src/approx_median.rs

```diff
@@ -57,20 +59,11 @@ make_udaf_expr_and_func!(
 ```"#,
     standard_argument(name = "expression",)
 )]
-#[derive(PartialEq, Eq, Hash)]
+#[derive(Debug, PartialEq, Eq, Hash)]
 pub struct ApproxMedian {
     signature: Signature,
 }
 
-impl Debug for ApproxMedian {
```

Comment:
I don't see much value in these manual implementations; I guess it adds the `name` field but I feel its better to just `derive` for simplicity. Happy to revert if desired.

Reply:
seems fine to me